### PR TITLE
N14: Bevy Engine

### PR DESCRIPTION
**Pull request by cart**
#278

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -66,6 +66,100 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [Bevy Engine][bevy]
+
+[![bevy logo](bevy_logo.png)](https://bevyengine.org/)
+
+[Bevy][bevy] is a refreshingly simple data-driven
```
Comment (kvark) [inline content/posts/newsletter-014/index.md:73]: I like this intro, but I think that maybe we don't need such a big introduction in every month where Bevy is mentioned? 1-2 sentences would be ideal :)

Comment (cart) [reply content/posts/newsletter-014/index.md:73]: Sure thing!

**Review by kvark — APPROVED** [review]

File: content/posts/newsletter-014/index.md [hunk 2]
```diff
@@ -66,6 +66,90 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+calculated LocalTransform and the source components and dataflow between components
+is hard to reason about. This problem was resolved by making a newer, simpler
+transform system that uses a consolidated `Transform` type.  
+- **Joystick/Gamepad Input**: The Bevy Input plugin now has cross-platform support
+for most controllers thanks to the gilrs library!
+- **Bevy ECS Performance Improvements**:
+  - *Generational Entity IDs*: Entity IDs have changed from being UUIDs to incrementing
+  generational indices. Random UUIDs were nice because they could be created anywhere,
+  were unique across game runs, and could be safely persisteto files or reused across
+  networks.Unfortunately they ended up being too slow relative to the alternatives.
```
Comment (kvark) [inline content/posts/newsletter-014/index.md:113]: Missing space before "Unfortunately"

File: content/posts/newsletter-014/index.md [hunk 3]
```diff
@@ -66,6 +66,90 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+are Good Reasons™ to use this approach, but it created a "lag" between the
+calculated LocalTransform and the source components and dataflow between components
+is hard to reason about. This problem was resolved by making a newer, simpler
+transform system that uses a consolidated `Transform` type.  
+- **Joystick/Gamepad Input**: The Bevy Input plugin now has cross-platform support
+for most controllers thanks to the gilrs library!
+- **Bevy ECS Performance Improvements**:
+  - *Generational Entity IDs*: Entity IDs have changed from being UUIDs to incrementing
+  generational indices. Random UUIDs were nice because they could be created anywhere,
+  were unique across game runs, and could be safely persisteto files or reused across
```
Comment (kvark) [inline content/posts/newsletter-014/index.md:112]: "persisteto" -> "persisted to"?

**Review by ozkriff — APPROVED** [review]
:+1: (Made a few style tweaks for consistency with the rest of the newsletter and trimmed some details from the updates overview)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**